### PR TITLE
fix #1162 duplicate image filename

### LINKED ISSUE #1162: Visual Regression Tests: BoundingBoxComputation: duplicate image filename
with this dirty(sorry... It is better to test for duplicate test names where more appropriate place) check:

660f488b33e71e8a3ff1484c448e2efeaddfc4fa: tests/vexflow_test_helpers.ts (patch to: 6dad8b20631745be97db32dcaac8772aec6e278f)
```diff
@@ -222,10 +222,14 @@ class VexFlowTests {
       // to test all fonts by default, we can remove this check.
       const onlyBravura = NODE_TEST_CONFIG.fontStacks.length === 1 && fontName === 'Bravura';
       const fontInfo = onlyBravura ? '' : `.${fontName}`;
       const fileName = `${VexFlowTests.NODE_IMAGEDIR}/${moduleName}.${testName}${fontInfo}.png`;
 
+      if (VexFlowTests.shims.fs.existsSync(fileName)) {
+        process.stderr.write(`duplicate fileName: ${fileName}\n`);
+      }
+
       const imageData = (element as HTMLCanvasElement).toDataURL().split(';base64,').pop();
       const imageBuffer = Buffer.from(imageData as string, 'base64');
 
       VexFlowTests.shims.fs.writeFileSync(fileName, imageBuffer, { encoding: 'base64' });
     }
```

Duplicate file names will be detected as follows:

```shell
$ npx grunt
$ rm -r ./build/images/current
$  npm run generate:current >& /tmp/l
$ grep duplicate /tmp/l | sort | uniq
BoundingBoxComputation :: Quadratic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Quadratic_Test.Bravura.png
BoundingBoxComputation :: Cubic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Cubic_Test.Bravura.png
BoundingBoxComputation :: Cubic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Cubic_Test.Bravura.png
BoundingBoxComputation :: Cubic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Cubic_Test.Gonville.png
BoundingBoxComputation :: Cubic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Cubic_Test.Petaluma.png
BoundingBoxComputation :: Quadratic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Quadratic_Test.Bravura.png
BoundingBoxComputation :: Quadratic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Quadratic_Test.Gonville.png
BoundingBoxComputation :: Quadratic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Quadratic_Test.Petaluma.png

# BoundingBoxComputation.Quadratic_Test.Bravura: 3+1 duplicateion:
$ grep ./build/images/current/BoundingBoxComputation.Quadratic_Test.Bravura.png /tmp/l
BoundingBoxComputation :: Quadratic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Quadratic_Test.Bravura.png
BoundingBoxComputation :: Quadratic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Quadratic_Test.Bravura.png
BoundingBoxComputation :: Quadratic Testduplicate fileName: ./build/images/current/BoundingBoxComputation.Quadratic_Test.Bravura.png

```

As a result, only the last item of the four tests (for example in `BoundingBoxComputation › Quadratic Test › Canvas + Bravura + flow.html`) will be generated in the visual test, as follows:

image

---

we need to modify the names of the tests to avoid duplicate file names.

### PR DESCRIPTION
**Pull request by rvilarl**
Fixes #1162 resolving TODO in test file.

### CODE REVIEW

File: tests/boundingboxcomputation_tests.ts [hunk 1]
```diff
@@ -16,8 +13,16 @@ const BoundingBoxComputationTests = {
     QUnit.module('BoundingBoxComputation');
     test('Point Test', point);
     const run = VexFlowTests.runTests;
-    quadraticParams.forEach((params /*, index*/) => run('Quadratic Test', quadratic, params));
-    cubicParams.forEach((params /*, index*/) => run('Cubic Test', cubic, params));
+    let n = 1;
+    quadraticParams.forEach((params /*, index*/) => {
```
Comment (ronyeh) [inline tests/boundingboxcomputation_tests.ts:17]: You should be able to uncomment the `/*, index*/` and use it instead of incrementing `n` manually. Array.forEach passes in the current item index.

File: tests/boundingboxcomputation_tests.ts [hunk 2]
```diff
@@ -16,8 +13,16 @@ const BoundingBoxComputationTests = {
...
+      run(`Quadratic Test ${n}`, quadratic, params);
+      n++;
+    });
+    n = 1;
+    cubicParams.forEach((params /*, index*/) => {
```
Comment (ronyeh) [inline tests/boundingboxcomputation_tests.ts:22]: Same as above.

**Review by ronyeh — CHANGES_REQUESTED** [review]

File: tests/boundingboxcomputation_tests.ts (hunk 1 above)
Comment (rvilarl) [reply tests/boundingboxcomputation_tests.ts]: done

**Review by ronyeh — APPROVED** [review]

**Comment by mscuthbert** [comment]
Good catch and cleanup all.
